info/Profile & Web/classic-editor.md

```diff
@@ -5,4 +5,4 @@ Last Updated: 2023-01-14 4:09 PM EST
 
 # Classic Web Editor
 
-We’re retiring our Classic Web Editor on January 31, 2023. [Read more here.](https://omglol.news/2023/01/14/so-long-classic-web-editor)
\ No newline at end of file
+We’re retiring our Classic Web Editor on January 31, 2023. [Read more here](https://omglol.news/2023/01/14/so-long-classic-web-editor).
```

Comment:
It's already retired, so maybe this page should be removed/redirected?

(disclaimer: i am not a staff.lol)